.github/workflows/benchmark.yml

```diff
@@ -93,15 +104,15 @@ jobs:
   generation_benchmark:
     needs: linux-build
     if: ${{github.event_name == 'schedule' || (!cancelled() && contains(fromJSON(github.event.inputs.benchmark_type), 'generation'))}}
-    runs-on: [self-hosted, linux-a100-2]
+    runs-on: [self-hosted, linux-a100]
```

Comment:
Is Linux-a100 142 server?

Reply:
No, run on 187.  currently only 4,5,6,7 Gpus used in benchmark for two runners, optimize cuda allocate to make each runner have balanced tasks.